srcpkgs/mozjs102/template

```diff
@@ -11,7 +11,7 @@ configure_args="--disable-jemalloc --disable-strip --disable-tests \
  --enable-shared-js --enable-system-ffi --with-intl-api --with-system-icu \
  --with-system-nspr --with-system-zlib --enable-hardening --enable-release"
 hostmakedepends="make pkg-config python3 python3-setuptools python3-six perl m4
- rust cargo llvm12 clang python3-looseversion"
+ rust cargo llvm18 clang18 python3-looseversion"
```

Comment:
For packages which only use llvm in hostmakedepends, and only because they want to use clang as the compiler, should we still use `llvm18 clang18` instead of `llvm clang`?